plugins/modules/ndo_tenant_igmp_interface_policy.py

```diff
@@ -0,0 +1,483 @@
+    - If both parameter are used, O(state_limit_route_map) will be ignored.
+    type: dict
+    suboptions:
+      name:
+        description:
+        - The name of the Route Map Policy for Multicast.
+        type: str
+        required: true
+    aliases: [ state_limit_route_map_policy, state_limit_route_map_policy_multicast ]
+  report_policy_route_map_uuid:
+    description:
+    - The UUID of the report policy route map.
+    type: str
+    aliases: [ report_policy_route_map_for_multicast ]
+  report_policy_route_map:
+    description:
+    - The Route Map Policy for Multicast.
+    - This parameter can be used instead of O(report_policy_route_map_uuid).
+    - If both parameter are used, O(report_policy_route_map) will be ignored.
+    type: dict
+    suboptions:
+      name:
+        description:
+        - The name of the Route Map Policy for Multicast.
+        type: str
+        required: true
+    aliases: [ report_policy_route_map_policy, report_policy_route_map_policy_multicast ]
+  static_report_route_map_uuid:
+    description:
+    - The UUID of the static report route map.
+    type: str
+    aliases: [ static_report_route_map_for_multicast ]
+  static_report_route_map:
+    description:
+    - The Route Map Policy for Multicast.
+    - This parameter can be used instead of O(static_report_route_map_uuid).
+    - If both parameter are used, O(static_report_route_map) will be ignored.
+    type: dict
+    suboptions:
+      name:
+        description:
+        - The name of the Route Map Policy for Multicast.
+        type: str
+        required: true
+    aliases: [ static_report_route_map_policy, static_report_route_map_policy_multicast ]
+  maximum_multicast_entries:
+    description:
+    - The maximum multicast entries value.
+    - Defaults to C(4294967295) when unset during creation.
+    - The valid range is from C(1) to C(4294967295).
+    - This parameter is only applicable when the O(state_limit_route_map_uuid) is not empty.
+    type: int
+  reserved_multicast_entries:
+    description:
+    - The reserved multicast entries value.
+    - Defaults to C(0) when unset during creation.
+    - The valid range is from C(0) to C(4294967295).
+    type: int
+  state:
+    description:
+    - Use C(absent) for removing.
+    - Use C(query) for listing an object or multiple objects.
+    - Use C(present) for creating or updating.
+    type: str
+    choices: [ absent, query, present ]
+    default: query
+notes:
+- The O(template) must exist before using this module in your playbook.
+  Use M(cisco.mso.ndo_template) to create the Tenant template.
+- The O(state_limit_route_map_uuid), O(report_policy_route_map_uuid), O(static_report_route_map_uuid) must exist before using this module in your playbook.
+  Use M(cisco.mso.ndo_tenant_route_map_policy_for_multicast) to create the Route Map Policy for Multicast.
+seealso:
+- module: cisco.mso.ndo_template
+- module: cisco.mso.ndo_tenant_route_map_policy_for_multicast
```

Comment:
ndo_tenant_route_map_policy_for_multicast is the wrong name. It think it's ndo_route_map_policy_multicast

Reply:
I would actually remove this